host/libhackrf/src/hackrf.h

```diff
@@ -66,6 +66,7 @@ enum hackrf_board_id {
 	BOARD_ID_JELLYBEAN  = 0,
 	BOARD_ID_JAWBREAKER = 1,
 	BOARD_ID_HACKRF_ONE = 2,
+	BOARD_ID_RAD1O = 3,
```

Comment:
I checked it yesterday and using `3` as board ID made `hackrf_info` recognize it correctly. I thought this was already specified, because [the wiki says so](https://github.com/mossmann/hackrf/wiki/libHackRF-API#supported-board-versions).

But I also got sceptical as you asked and looked in the rad1o fork of hackrf and they definitively use board ID of 3: https://github.com/rad1o/hackrf/blob/rad1o/firmware/common/hackrf_core.h#L39. They even have an issue (rad1o/hackrf#9) for submitting it upstream ;)

AFAIK they even use both (you can choose the mode a.k.a. `hackrf` vs. `hackrf-old` on startup to be compatible with apps which do not yet support it but the HackRF).